### PR TITLE
92 | Adds binder links to run notebooks against specific tagged versions

### PR DESCRIPTION
**Pull request by ns-rse**
Following the examples in [this post](https://blog.jupyter.org/binder-2-0-a-tech-guide-2017-fd40515a3a84) it was quite easy to modify the Binder links to run against tagged versions.

This might be useful for future development as you can run them against specific commits, just replace the relevant part of the URL.

### CODE REVIEW

**Comment by bobturneruk** [comment]
Can you make it clearer which one is the latest / default version, please?

**Review by bobturneruk — APPROVED** [review]
I think this is good. I've dropped @smesnage a line to see what he thinks. Please hold off for a response on that before merging.

**Comment by ns-rse** [comment]
Merged after the thumbs up (via email) from @smesnage (although there appear to be problems with binder, see #98 which are being investigated separately).

**Pull request merged**
